rfcs/0000-c-unwind-abi.md

```diff
@@ -57,11 +51,11 @@ several requirements for any cross-language unwinding design.
 The ["Analysis of key design goals"][analysis-of-design-goals] section analyzes
 how well the current design satisfies these constraints.
 
-* **Changing from panic=unwind to panic=abort cannot cause UB:** We
-  wish to ensure that choosing `panic=abort` doesn't ever create
-  undefined behavior (relate to `panic=unwind`), even if one is
-  relying on a library that triggers a panic or a foreign exception.
-* **Optimization with panic=abort:** when using `-Cpanic=abort`, we
+* **Changing from `panic=unwind` to `panic=abort` cannot cause undefined
+  behavior:** We wish to ensure that choosing `panic=abort` doesn't ever create
+  undefined behavior (relate to `panic=unwind`), even if one is relying on a
```

Comment:
It's not immediately clear to me what this parenthetical means. Maybe "changing from `panic=unwind` to `panic=abort` never creates undefined behavior", if that's the intent?

Reply:
Agreed that phrasing is clearer

Reply:
Oops, I think I copied this bullet from a conversation somewhere and didn't read it closely. Glad that adding the backticks happened to draw attention to it!